src/CMakeLists.txt

```diff
@@ -3,22 +3,25 @@ cmake_minimum_required(VERSION 2.8)
 list(APPEND CMAKE_MODULE_PATH ${CIME_CMAKE_MODULE_DIRECTORY})
 include(CIME_initial_setup)
 
-project(clm45_tests Fortran C)
+#list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../tools/mksurfdata_esmf/cmake")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../share/cmake")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../component/cmeps/cmake")
+
+project(clm_tests Fortran C)
 
 include(CIME_utils)
 
 set(CLM_ROOT "..")
 
-# This definition is needed to avoid having ESMF depend on mpi
-add_definitions(-DHIDE_MPI)
+#find_package(ESMF 8.2.0 REQUIRED )
```

Comment:
I see that this is commented out. How do you get away without needing ESMF, now that we no longer have the esmf wrf timemgr?

Reply:
Good question. ESMF comes in by including the share code which adds this there. The problem I had was that having this explicitly here conflicted with the

`
add_library(ESMF)`

in the share code cmake. So I needed to remove it here.

I'll add a comment about this to point this out...

Reply:
Sounds good - thanks for clarifying. Yes, a comment about that would be great - thanks!

Reply:
Added some comments, so resolving.